api/nodes/views.py

```diff
@@ -1660,7 +1662,14 @@ def get_queryset(self):
 
     # overrides ListCreateAPIView
     def perform_create(self, serializer):
-        serializer.save(node=self.get_node())
+        user = get_user_auth(self.request).user
+        node = self.get_node()
+        try:
+            fork = serializer.save(node=node)
+            mails.send_mail(user.email, mails.FORK_COMPLETED, title=fork.title, guid=fork._id, mimetype='html')
+        except Exception as exc:
+            mails.send_mail(user.email, mails.FORK_FAILED, title=node.title, guid=fork._id, mimetype='html')
```

Comment:
I think guid needs to be `node._id` since to get here fork would have to fail to save. Did it work locally if you raise an error in serializer?